examples/client/react/src/main.tsx

```diff
@@ -15,6 +15,7 @@ async function getToken() {
     access: _access,
   })
   if (next.err) return
+  if (!next.tokens) return _access
```

Comment:
handling this case https://github.com/openauthjs/openauth/blob/c87f0a78985d65ac7583f69b7abfe2583d1277a6/packages/openauth/src/client.ts#L146-L148